src/Data/List/Relation/Unary/Unique/Setoid/Properties.agda

```diff
@@ -156,3 +158,21 @@ module _ (S : Setoid a ℓ) {P : Pred _ p} (P? : Decidable P) where
 
   filter⁺ : ∀ {xs} → Unique S xs → Unique S (filter P? xs)
   filter⁺ = AllPairs.filter⁺ P?
+
+------------------------------------------------------------------------
+-- ∷
+
+module _ (S : Setoid a ℓ) where
+
+  open Setoid S renaming (Carrier to A)
+  open Membership S using (_∈_; _∉_)
+
+  private variable x y : A; xs : List A
```

Comment:
As with the other (excellent!) introductions of local `private variable` declarations, suggest spacing this out as:
```suggestion
  private
    variable
      x y : A
      xs : List A
```